playbook/app/pb_kits/playbook/pb_caption/_caption_mixin.scss

```diff
@@ -39,3 +42,10 @@ $pb_dark_caption_colors: (
 @mixin caption_dark {
   color: $text_dk_light;
 }
+@mixin pb_caption_kit_colors {
+  @each $name, $color in $pb_caption_colors {
+    &[class*="#{$name}"] {
```

Comment:
@jasperfurniss `pb_caption_kit_colors` gets imported above where is a breaking change. Do you think it's `_` that we're missing before name here?